src/mpc-hc/FGManager.cpp

```diff
@@ -2580,7 +2580,7 @@ CFGManagerPlayer::CFGManagerPlayer(LPCTSTR pName, LPUNKNOWN pUnk, HWND hWnd, boo
                 break;
         }
     } else {
-        if (CAppSettings::IsVideoRendererAvailable(VIDRNDT_DS_EVR)) {
+        if (false && CAppSettings::IsVideoRendererAvailable(VIDRNDT_DS_EVR)) {

```

Comment:
Forgot to remove

Reply:
Yes, I had removed it, but most not have saved before commit.